examples/getST.py

```diff
@@ -533,6 +583,44 @@ def doS4U(self, tgt, cipher, oldSessionKey, sessionKey, nthash, aesKey, kdcHost)
         authenticator['cusec'] = now.microsecond
         authenticator['ctime'] = KerberosTime.to_asn1(now)
 
+        tgsReq = TGS_REQ()
+
+        reqBody = seq_set(tgsReq, 'req-body')
+
+        opts = list()
+        # This specified we're doing S4U
+        opts.append(constants.KDCOptions.cname_in_addl_tkt.value)
+        opts.append(constants.KDCOptions.canonicalize.value)
+        opts.append(constants.KDCOptions.forwardable.value)
+        opts.append(constants.KDCOptions.renewable.value)
+
+        reqBody['kdc-options'] = constants.encodeFlags(opts)
+        service2 = Principal(self.__options.spn, type=constants.PrincipalNameType.NT_SRV_INST.value)
+        seq_set(reqBody, 'sname', service2.components_to_asn1)
+        reqBody['realm'] = self.__domain
+
+        myTicket = ticket.to_asn1(TicketAsn1())
+        seq_set_iter(reqBody, 'additional-tickets', (myTicket,))
+
+        now = datetime.datetime.utcnow() + datetime.timedelta(days=1)
+
+        reqBody['till'] = KerberosTime.to_asn1(now)
+        reqBody['nonce'] = random.getrandbits(31)
+        seq_set_iter(reqBody, 'etype', (int(cipher.enctype), ))
+
+        reqBodyEncCksumType = cksumtype_for_enctype(cipher.enctype)
+
+        reqBodyEnc = encoder.encode(reqBody)[4:]
```

Comment:
Same issue here, but this time the 4 first bytes are irrelevant for computing the KDC-REQ-BODY checksum.